packages/website/content/docs/overview/getting-started.md

```diff
@@ -42,47 +46,82 @@ In the same `src/App.tsx` file, use `InefficientComponent`:
       </InefficientComponent>
 ```
 
-4. Modify Build Configuration
+### 4. Modify build configuration
 
-In the app's root directory, modify `webpack.config.js`:
+In order to generate the flamegraphs for the app, we need to build the app for production. Normally, we'd simply
+run `yarn build` which is an alias for `react-scripts build`, which in turn, under-the-hood calls `webpack` to
+optimize and minify the app. However, for the flamegraphs to be readable and meaningful, we need to disable the
+minification. While `react-scripts` don't allow us to tweak the `webpack` out of the box, it is possible via
+`react-app-rewired` package. With `react-app-rewired` added as a dev dependency, we can tell it to override certain
+configuration values of `webpack` such as turning minification off for the production build.
```

Comment:
I worry this is straying to far from the concerns of a basic getting started example. I would rather leave it as it was or create a similarly simple example without using `react-scripts`, if these additional steps are unavoidable.

Reply:
Actually, that sounds good! I'm more than happy to add an additional section to the guide, "Examples" say, or something. The main point of this is for users who do use the modern `react-scripts` approach to guide them how to work with Flamegrill to get their flamegraphs out. How does that sound to you?

Reply:
OK, so I've now reshuffled things a little bit and reverted back the original quick start guide to what it was, and added a new section to  the docs called `Examples`, and moved the fluentui-cra-template-app example there. Have a look and lemme know what you reckon!